core/src/main/scala/dagr/core/tasksystem/Pipe.scala

```diff
@@ -118,6 +118,12 @@ object Pipes {
     this.requires(cores, memory)
   }
 
+  /** A type alias fdagr.core.tasksystemor sink tasks that can receive data via a pipe, but cannot pipe onwards. */
+  type PipeIn[In] = dagr.core.tasksystem.Pipe[In,Void]
+
+  /** A type alias for generate tasks that can pipe out, but cannot receive data from a pipe. */
+  type PipeOut[Out] = dagr.core.tasksystem.Pipe[Void,Out]
+
```

Comment:
@tfenne check this out.  Since I cannot access `Void` outside of here (that's good), but my method would return `Pipe[Void,Out]`, I need this alias.  I can't use `PipeOut[Out]`.